.github/workflows/main.yml

```diff
@@ -12,11 +12,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - run: yarn
-    - run: yarn lint
-    - uses: gozala/typescript-error-reporter-action@v1.0.8
```

Comment:
Did this get pulled out intentionally? It is quite useful.

Reply:
I changed based on: https://github.com/ipfs/aegir/blob/master/md/github-actions.md

Thought the recommendation was `ts -p check`

Reply:
The error reporter action shows you which lines have errors in a visual form as part of the PR, the aegir command makes you dig through the console output.  Both do the same thing and use the same rules but the action is a little more user friendly IMHO.